framework/CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ Yii Framework 2 Change Log
 - Bug #18988: Fix default value of `yii\console\controllers\MessageController::$translator` (WinterSilence)
 - Bug #18993: Load defaults by `attributes()` in `yii\db\ActiveRecord::loadDefaultValues()` (WinterSilence)
 - Bug #19021: Fix return type in PhpDoc `yii\db\Migration` functions `up()`, `down()`, `safeUp()` and `safeDown()` (WinterSilence, rhertogh)
+- Bug #19031: Refactor `yii\console\Controller::getActionArgsHelp()` (WinterSilence)
```

Comment:
```suggestion
- Bug #19031: Fix displaying console help for parameters with declared types (WinterSilence)
```

Reply:
@samdark also, detect type by default value https://github.com/yiisoft/yii2/pull/19031/files#diff-0668ca3525fab06caf4ec3a07ef948398c018944220976c38e94d91f0dfbbb35R598 and analyze not complete PHPDoc

Reply:
@samdark we agreed not to change anything in my PR without my consent